### PR TITLE
Upgrade terraform-provider-kafka to v0.5.3

### LINKED ISSUE #145: Upgrade terraform-provider-kafka to v0.5.3
Release details: https://github.com/Mongey/terraform-provider-kafka/releases/tag/v0.5.3

### PR DESCRIPTION
**Pull request by guineveresaenger**
This PR was generated via `$ upgrade-provider pulumi-kafka`.

---

Upgrading terraform-provider-kafka from 0.5.2 to 0.5.3.
Upgrading pulumi-terraform-bridge from v3.38.1 to v3.44.3.

Fixes #145

### CODE REVIEW

File: provider/resources.go [hunk 1]
```diff
@@ -148,10 +153,30 @@ func Provider() tfbridge.ProviderInfo {
...
+	}
+
+	err = x.AutoAliasing(&prov, prov.GetMetadata())
+	contract.AssertNoErrorf(err, "auto aliasing apply failed")
+
 	prov.SetAutonaming(255, "-")
 
 	return prov
 }
+
```
Comment (iwahbe) [inline provider/resources.go:181]: This needs a doc comment using `go:embed`.
```suggestion
//go:embed cmd/pulumi-resource-kafka/bridge-metadata.json
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -311,131 +311,14 @@ jobs:
...
-      name: Publish SDKs
-      run: ./ci-scripts/ci/publish-tfgen-package ${{ github.workspace }}
-    - if: failure() && github.event_name == 'push'
-      name: Notify Slack
-      uses: 8398a7/action-slack@v3
-      with:
-        author_name: Failure in publishing SDK
-        fields: repo,commit,author,action
-        status: ${{ job.status }}
+      - uses: actions/checkout@v3
```
Comment (aq17) [inline .github/workflows/release.yml:319]: 😄

Comment (guineveresaenger) [reply .github/workflows/release.yml:319]: that was on purpose, to test this action.

**Comment by guineveresaenger** [comment]
This update is BLOCKED until https://github.com/Mongey/terraform-provider-kafka/issues/329 is resolved.

File: .github/workflows/release.yml (hunk 2 above)
Comment (t0yv0) [reply .github/workflows/release.yml:319]: I figured. Thank you.

**Pull request closed without merging**